.ci.yaml

```diff
@@ -3215,6 +3215,7 @@ targets:
 
   - name: Mac_ios hot_mode_dev_cycle_macos_target__benchmark
     recipe: devicelab/devicelab_drone
+    presubmit: false
```

Comment:
can you link https://github.com/flutter/flutter/issues/94946 in a comment?

Reply:
Added 😃